_posts/platform/deployment/2000-01-01-deploy-with-git.md

```diff
@@ -85,3 +85,56 @@ It will also add the `scalingo` remote to simplify future pushes.
   in any other way (GitLab / GitHub integration, archive, ...) the changes made
   to your code won't be available.
 {% endnote %}
+
+
+### Shallow Error
+
+Currently, Scalingo does not support update of shallowed repository (i.e. from a `git push`).
+The `shallow error` means that the pushed repository on Scalingo does not contains all its history.
```

Comment:
You write `shallow error`as if it's a quote of an error message. But it does not look like a quote of an error message, is it?

Reply:
Yes you're right